repos/system_upgrade/common/actors/scanfilestobecopiedintotargetuserspace/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.scanfilestobecopiedintotargetuserspace import scan_files_to_copy
+from leapp.models import CopyFile
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class ScanFilesToBeCopiedIntoTargetUserspace(Actor):
+    """
+    Scan the source system and identify files that will be copied into the target userspace when it is created.
+    """
+
+    name = 'scan_files_to_be_copied_into_target_userspace'
+    consumes = ()
+    produces = (CopyFile,)
```

Comment:
This does not have the expected affect. The `CopyFile` is just a model that is not consumed by any actor and it is expected to be used just as part of other models in the file - in this case, it is expected to produce `TargetUserSpacePreupgradeTasks` : so something like `TargetUserSpacePreupgradeTasks(copy_files=[CopyFile(....)])`

Reply:
The requested changes are implemented in the latest version :smiley_cat: .

Reply:
@pirat89 Aah, thanks for catching this. I was under impression it was ok after the refactoring. Makes sense now.